examples/quantumgames/example_extended_nonlocal_games.py

```diff
@@ -0,0 +1,716 @@
+"""Extended nonlocal games
+==========================
+
+In this tutorial, we will define the concept of an *extended nonlocal game*.
+Extended nonlocal games are a more general abstraction of nonlocal games
+wherein the referee, who previously only provided questions and answers to the
+players, now share a state with the players and is able to perform a
+measurement on that shared state.
+"""
+# sphinx_gallery_thumbnail_path = '../../figures/extended_non_local_game.svg'
```

Comment:
I do not think this thumbnail is working. 

![image](https://github.com/user-attachments/assets/3e97eff9-0df9-4d68-8549-4f31725551d6)